src/python/pants/notes/1.25.x.rst

```diff
@@ -0,0 +1,563 @@
+1.25.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.25.x`` ``stable`` series.
+
+The ``1.25.x`` series brings two major changes to Pants:
+
+1. Deprecating ``globs()``, ``rglobs()``, and ``zglobs()`` in BUILD files.
+2. Adding file args.
+
+Please see https://groups.google.com/forum/#!topic/pants-devel/3nmdSeyvwU0 for more information.
+
+1.25.0rc0 (02/4/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Deprecate `globs`, `rglobs`, and `zglobs` (#9040)
+  `PR #9040 <https://github.com/pantsbuild/pants/pull/9040>`_
+
+* Deprecate `--owner-of` in favor of file arguments (#9050)
+  `PR #9050 <https://github.com/pantsbuild/pants/pull/9050>`_
+
+* Deprecate `--build-file-imports` defaulting to `warn` and having the option `allow` (#9047)
+  `PR #9047 <https://github.com/pantsbuild/pants/pull/9047>`_
+
+* Deprecate `--glob-expansion-failure` in favor of `--files-not-found-behavior` (#9022)
+  `PR #9022 <https://github.com/pantsbuild/pants/pull/9022>`_
+
+* [IntelliJ] Add compiler options and plugins to the output of `export-dep-as-jar` (#9036)
+  `PR #9036 <https://github.com/pantsbuild/pants/pull/9036>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow using file args with every goal (#9048)
+  `PR #9048 <https://github.com/pantsbuild/pants/pull/9048>`_
+
+* Support python2 in the setup.py rule. (#8956)
+  `PR #8956 <https://github.com/pantsbuild/pants/pull/8956>`_
+
+Bugfixes
+~~~~~~~~
+
+* Fix Python linters to work without `pants.backend.python` registered (#9059)
+  `PR #9059 <https://github.com/pantsbuild/pants/pull/9059>`_
+
+* [thrift linter] use non-zero exit on warning when strict and zero exit warning when not strict (#9051)
+  `PR #9051 <https://github.com/pantsbuild/pants/pull/9051>`_
+
+* Fix V2 `test` and `setup-py2` to work with file args (#9056)
+  `PR #9056 <https://github.com/pantsbuild/pants/pull/9056>`_
+
+* Fix race condition with deprecation warnings (#9060)
+  `PR #9060 <https://github.com/pantsbuild/pants/pull/9060>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Error when literal file args do not have any owners (#9068)
+  `PR #9068 <https://github.com/pantsbuild/pants/pull/9068>`_
+
+* Change invalid file args to error instead of warn (#9041)
+  `PR #9041 <https://github.com/pantsbuild/pants/pull/9041>`_
+
+* Preserve the filesystem spec used to find owning targets (#9063)
+  `PR #9063 <https://github.com/pantsbuild/pants/pull/9063>`_
+
+* Decouple `--owner-of` implementation from` --changed-*` (#9046)
+  `PR #9046 <https://github.com/pantsbuild/pants/pull/9046>`_
+
+* Remove deps from the core into the Python backend's subsystems (#9049)
+  `PR #9049 <https://github.com/pantsbuild/pants/pull/9049>`_
+
+* Allow options to be registered passively. (#9061)
+  `PR #9061 <https://github.com/pantsbuild/pants/pull/9061>`_
```

Comment:
This is a new feature for us Pants devs, but I don't think it's supposed to impact end users.

Reply:
Correct